neatplus/views.py

```diff
@@ -31,3 +37,33 @@ def fetch_related_objects(self, queryset):
                     if field in expand_params or wildcard_present:
                         queryset = queryset.select_related(field)
         return queryset
+
+
+def ckeditor_file_upload_view(request):
+    if request.method == "POST" and "upload" in request.FILES:
+        # Choose the storage backend dynamically
+        if os.getenv("USE_S3_STORAGE", "false").lower() == "true":
+            storage = CKEditorStorage()
+        else:
+            storage = FileSystemStorage()  # Save to local MEDIA_ROOT
```

Comment:
use from django.core.files.storage import default_storage to get current storage